django_migration_testcase/base.py

```diff
@@ -74,9 +74,10 @@ def run_reverse_migration(self):
         raise NotImplementedError()
 
     def migrate_kwargs(self):
-        return {'verbosity': 0,
-                'no_initial_data': True,
```

Comment:
Anyone has any idea why `no_initial_data` was there? Compatibility for X version of django, South or something?